package.json

```diff
@@ -56,6 +56,7 @@
         "bundlesize": "^0.17.0",
         "doctoc": "^1.4.0",
         "eslint": "^5.16.0",
+        "eslint-plugin-dcr": "https://github.com/guardian/eslint-plugin-dcr",
```

Comment:
It would be nice to tag this library and then depend on the specific tagged version, as it would give us a little bit more safety around changes to the library. But I'm not very concerned as this dep runs at build time not prod.

Reply:
Ah yeah good idea